rdsbroker/broker_test.go

```diff
@@ -1637,6 +1638,43 @@ var _ = Describe("RDS Broker", func() {
 			})
 		})
 
+		Context("when using 'read_only' custom parameters", func() {
+			BeforeEach(func() {
+				allowUserBindParameters = true
+			})
+
+			It("when database engine is 'postgres'", func() {
+				bindDetails.RawParameters = json.RawMessage(`{"read_only": true}`)
+				_, err := rdsBroker.Bind(ctx, instanceID, bindingID, bindDetails, false)
+				Expect(err).ToNot(HaveOccurred())
+			})
+
+			Context("when database engine is 'mysql'", func() {
+				BeforeEach(func() {
+					rdsInstance.DescribeReturns(
+						&rds.DBInstance{
+							Engine: aws.String("mysql"),
+						},
+						nil,
+					)
+				})
+
+				It("read_only eq true", func() {
+					bindDetails.RawParameters = json.RawMessage(`{"read_only": true}`)
+					_, err := rdsBroker.Bind(ctx, instanceID, bindingID, bindDetails, false)
+					Expect(err).To(HaveOccurred())
```

Comment:
non-blocking: consider breaking out the error into a `var` just like `ErrEncryptionNotUpdateable` so the test can check it explicitly and avoid this this passing due to something else returning an error

```suggestion
					Expect(err).To(MatchError(ErrReadOnlyNotSupported))
```